config-templates/aws/aws-ebs-csi-driver/1.12.0/deployment.yaml

```diff
@@ -0,0 +1,637 @@
+        app.kubernetes.io/name: aws-ebs-csi-driver
+        app.kubernetes.io/version: 1.12.0
+    subjects:
+      - kind: ServiceAccount
+        name: ebs-csi-controller-sa
+        namespace: kube-system
+    roleRef:
+      kind: ClusterRole
+      name: ebs-external-resizer-role
+      apiGroup: rbac.authorization.k8s.io
+  - kind: ClusterRole
+    apiVersion: rbac.authorization.k8s.io/v1
+    metadata:
+      name: ebs-external-snapshotter-role
+      labels:
+        app.kubernetes.io/name: aws-ebs-csi-driver
+        app.kubernetes.io/version: 1.12.0
+    rules:
+      - apiGroups: [""]
+        resources: ["events"]
+        verbs: ["list", "watch", "create", "update", "patch"]
+      - apiGroups: [""]
+        resources: ["secrets"]
+        verbs: ["get", "list"]
+      - apiGroups: ["snapshot.storage.k8s.io"]
+        resources: ["volumesnapshotclasses"]
+        verbs: ["get", "list", "watch"]
+      - apiGroups: ["snapshot.storage.k8s.io"]
+        resources: ["volumesnapshotcontents"]
+        verbs: ["create", "get", "list", "watch", "update", "delete"]
+      - apiGroups: ["snapshot.storage.k8s.io"]
+        resources: ["volumesnapshotcontents/status"]
+        verbs: ["update"]
+  - kind: ClusterRoleBinding
+    apiVersion: rbac.authorization.k8s.io/v1
+    metadata:
+      name: ebs-csi-snapshotter-binding
+      labels:
+        app.kubernetes.io/name: aws-ebs-csi-driver
+        app.kubernetes.io/version: 1.12.0
+    subjects:
+      - kind: ServiceAccount
+        name: ebs-csi-controller-sa
+        namespace: kube-system
+    roleRef:
+      kind: ClusterRole
+      name: ebs-external-snapshotter-role
+      apiGroup: rbac.authorization.k8s.io
+  - kind: DaemonSet
+    apiVersion: apps/v1
+    metadata:
+      name: ebs-csi-node
+      namespace: kube-system
+      labels:
+        app.kubernetes.io/name: aws-ebs-csi-driver
+        app.kubernetes.io/version: 1.12.0
+    spec:
+      selector:
+        matchLabels:
+          app: ebs-csi-node
+          app.kubernetes.io/name: aws-ebs-csi-driver
+          app.kubernetes.io/version: 1.12.0
+      updateStrategy:
+        type: RollingUpdate
+        rollingUpdate:
+          maxUnavailable: "10%"
+      template:
+        metadata:
+          labels:
+            app: ebs-csi-node
+            app.kubernetes.io/name: aws-ebs-csi-driver
+            app.kubernetes.io/version: 1.12.0
+          annotations:
+            autoUpdate: "AUTO-UPDATE-ENABLED"  
+        spec:
+          affinity:
+            nodeAffinity:
+              requiredDuringSchedulingIgnoredDuringExecution:
+                nodeSelectorTerms:
+                - matchExpressions:
+                  - key: eks.amazonaws.com/compute-type
+                    operator: NotIn
+                    values:
+                    - fargate
+          nodeSelector:
+            kubernetes.io/os: linux
+          serviceAccountName: ebs-csi-node-sa
+          priorityClassName: system-node-critical
```

Comment:
@gauravAggrhari Old plugin versions have `system-cluster-critical` instead of `system-node-critical`. Please check
https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/aws/aws-ebs-csi-driver/1.5.1/deployment.yaml#L258

Reply:
I think the latest driver has this only. @mssachan